dashboards/lodestar_networking.json

```diff
@@ -534,9 +538,10 @@
           "values": false
         },
         "showUnfilled": true,
-        "text": {}
+        "text": {},
+        "valueMode": "color"
       },
-      "pluginVersion": "9.3.2",
+      "pluginVersion": "10.1.1",
```

Comment:
Was curious if we can also get rid of `pluginVersion` but it seems to be required for panel migrations when the version is different (https://github.com/grafana/grafana/issues/76419#issuecomment-1759720333)